pkgs/by-name/go/gomuks-web/package.nix

```
@@ -0,0 +1,73 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildGo123Module,
```

```suggestion
  buildGoModule,
```

We can unpin go, since upstream pins 1.24.1 as the toolchain.